plone/scale/storage.py

```diff
@@ -14,6 +14,15 @@
 # See AnnotationStorage._keep_scale_seconds instead.
 KEEP_SCALE_MILLIS = 24 * 60 * 60 * 1000
 
+try:
+    long
+except NameError:
+    # Python 3 has no long, only int.
```

Comment:
With javascript code we are used to checking for features, not versions, so that's what I did here. I don't know what's usual here. Requiring `six` and using `six.integer_types` is also an option.

I don't think this package is Python3 ready, but that is just a guess. I simply remembered reading not so long ago about `long` not being defined in Python 3, and I did not want to introduce a new incompatibility.